pyproject.toml

```diff
@@ -127,6 +127,7 @@ extend-exclude = [
 'src/trio/lowlevel.py' = ['F401']
 'src/trio/socket.py' = ['F401']
 'src/trio/testing/__init__.py' = ['F401']
+'**/type_tests/*' = ['F841']  # Annotated local vars are used to specify return values.
```

Comment:
Is this still necessary? (should we be using `typing.assert_type` instead?)

Reply:
In this case I was deliberately testing `start()` calls with/without the return type explicitly defined, to see if Mypy could infer them. It's not necessary right now after I removed that though.

Reply:
```suggestion
```